async-http-client/src/main/scala/org/http4s/client/asynchttpclient/AsyncHttpClient.scala

```diff
@@ -188,8 +190,18 @@ object AsyncHttpClient {
     val empty: java.util.List[Cookie] = new java.util.ArrayList()
     override def add(uri: Uri, cookie: Cookie): Unit = ()
     override def get(uri: Uri): java.util.List[Cookie] = empty
-    override def getAll(): java.util.List[Cookie] = empty
+    override def getAll: java.util.List[Cookie] = empty
     override def remove(pred: java.util.function.Predicate[Cookie]): Boolean = false
     override def clear(): Boolean = false
   }
+
+  final case class AsyncHttpClientStats[F[_]: Sync](private val underlying: ClientStats)(implicit
```

Comment:
I'd put this class in its own source file.

Reply:
Moved to it's own file